zowe/zowe-cli
# Add Apple Silicon support for Zowe daemon

**What It Does**
Resolves #1764

**How to Test**
* Download the MacOS artifact from this PR's workflow run: https://github.com/zowe/zowe-cli/actions/runs/5684509878
* Extract the TGZ from the downloaded ZIP and rename it to "zowe-macos.tgz"
* Navigate to the directory where Zowe CLI is installed and replace the TGZ in the "prebuilds" folder
* Run `zowe daemon enable` followed by a CLI command to verify that the new daemon works

**Review Checklist**
I certify that I have:
  - [x] Test on ARM Mac - tested on my local machine
  - [x] Test on x86_64 Mac - tested in [this workflow run](https://github.com/zowe/zowe-cli/actions/runs/5717741634)

**Additional Comments**
The universal binary is about twice as large as the old one, but I think the benefits make it worthwhile.
